### PR TITLE
My Comment Reactions

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
MY COMMENT REACTIONS/REPLIES | As a commenter, I want to see in My Profile how many likes/reactions and replies my comments have received so that I can know if I want to revisit the comment section

https://www.pivotaltracker.com/story/show/150595787

### CODE REVIEW

**Comment by kgardnr** [comment]
Hey @okbel the only thing is to singularize "reactions" and "replies" when there's only 1: 
my_reactions_replies

**Comment by alinhle** [comment]
Also it should be hidden when the count is 0.

File: client/coral-framework/utils/index.js
Comment (alinhle) [inline client/coral-framework/utils/index.js:190]: This is not a reliable method to determine `reactions`. There is not only `FlagActionSummary` but also `DontAgreeActionSummary` and on top of that not all plugin action summaries are sure to be reactions. What's your stance on this @wyattjoh?

**Comment by kgardnr** [comment]
That's a good idea @cvle, definitely reduces clutter to the eye as well - @IAmSamHankins has the 0 counts showing, but we can chat with her later.

**Comment by alinhle** [comment]
I think it was a requirement from @losowsky but maybe not? :-D Anyway I think it's a good idea.

**Comment by alinhle** [comment]
I'd also add more space between _reactions_ and _replies_ @IAmSamHankins to have that more visually separated.

**Comment by okbel** [comment]
We decided that if the count was 0 it should still be shown. We can re-check this. @IAmSamHankins  thoughts?

File: client/coral-framework/utils/index.js
Comment (okbel) [reply client/coral-framework/utils/index.js:190]: Seems fragile for me too as well. But I can't see other filtering option based on this structure. *Interested in what Wyatt has to say about this*

**Comment by theysamtheirs** [comment]
@okbel, yes we should still show the counts even if zero. to address "clutter" issue, let's have anything displaying 0 with a color of #9E9E9E (lighter for zero, so as to be less distracting and also draw more attention when there are replies or reactions.)

**Comment by okbel** [comment]
@IAmSamHankins Ready to be checked! :)

**Comment by okbel** [comment]
Styles and translations added. // @kgardnr

**Comment by kgardnr** [comment]
Awesome, thanks @okbel!

@IAmSamHankins is this good to go then?

reactions_replies_take_two

**Comment by theysamtheirs** [comment]
LGTM!

**Comment by kgardnr** [comment]
Cool will just wait for @wyattjoh's feedback before merging.

File: client/coral-framework/utils/index.js
Comment (wyattjoh) [reply client/coral-framework/utils/index.js:190]: I think we could use a GraphQL union type? But I'm a bit unsure..

Comment (wyattjoh) [reply client/coral-framework/utils/index.js:190]: For now we can leave this as is. This might be a bigger task to classify all reaction types properly.

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**
